src/libraries/sendtohelixhelp.proj

```diff
@@ -233,13 +232,46 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <_ShouldSetXUnitLogCheckerEnvVar>0</_ShouldSetXUnitLogCheckerEnvVar>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsXUnitLogCheckerSupported)' == 'true'">
+    <_ShouldSetXUnitLogCheckerEnvVar Condition="$(TargetFrameworkIdentifier) == '.NETCoreApp'">1</_ShouldSetXUnitLogCheckerEnvVar>
+    <HelixPreCommands Condition="'$(TargetOS)' == 'windows'">
+      $(HelixPreCommands);
+      set __TestArchitecture=$(TargetArchitecture)
+    </HelixPreCommands>
+    <HelixPreCommands Condition="'$(TargetOS)' != 'windows'">
+      $(HelixPreCommands);
+      export __TestArchitecture=$(TargetArchitecture)
+    </HelixPreCommands>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- The IsXUnitLogCheckerSupported property should always be set. -->
```

Comment:
Hmm I thought I left a comment for this. Why do you need the env var to always be defined? That's not what we usually do.

Reply:
I want to skip the execution of XUnitLogChecker in the lines below in the script when it's value is set to zero.

Reply:
Right, but why can't you skip execution when the env var isn't defined or 0?

Reply:
Okay yes, I can move them up here.